pages/mod/_id/version/_version/index.vue

```diff
@@ -345,6 +345,15 @@ export default {
     }
   }
 
+  .accent-button {
```

Comment:
This may be used in other contexts, could it be moved to the components.scss file?